src/components/signup/SignupState.js

```diff
@@ -4,7 +4,7 @@ import { AsyncStorage, ScrollView, StyleSheet, View } from 'react-native'
 import { createSwitchNavigator } from '@react-navigation/core'
 import { isMobileSafari } from 'mobile-device-detect'
 import _get from 'lodash/get'
-import { GD_USER_MNEMONIC, IS_LOGGED_IN } from '../../lib/constants/localStorage'
+import { GD_USER_MASTERSEED, GD_USER_MNEMONIC, IS_LOGGED_IN } from '../../lib/constants/localStorage'
```

Comment:
about changes for torus looks ok
but generally this component looks very very overloaded.
we need to review it's code and split - added #1530 task

Reply:
ok